spark-doris-connector/src/main/scala/org/apache/doris/spark/sql/SchemaUtils.scala

```diff
@@ -166,13 +166,16 @@ private[spark] object SchemaUtils {
       case dt: DecimalType => row.getDecimal(ordinal, dt.precision, dt.scale)
       case at: ArrayType =>
         val arrayData = row.getArray(ordinal)
-        var i = 0
-        val buffer = mutable.Buffer[Any]()
-        while (i < arrayData.numElements()) {
-          if (arrayData.isNullAt(i)) buffer += null else buffer += rowColumnValue(arrayData, i, at.elementType)
-          i += 1
-        }
-        s"[${buffer.mkString(",")}]"
+        val result: String = Option(arrayData)
```

Comment:
🎉Thank you for your contribution! 🙌 It is necessary to consider the case where the array is empty (size = 0). The code can be further improved! 

```
        if (arrayData == null) DataUtil.NULL_VALUE else {
          if (arrayData.numElements() == 0) "[]" else {
            (0 until arrayData.numElements()).map(i => {
              if (arrayData.isNullAt(i)) null else rowColumnValue(data, i, at.elementType)
            }).mkString("[", ",", "]")
          }
        }

```

Reply:
@wolfboys Thank you for reminding me. You are right. I will improve it